packages/mds-types/device.ts

```diff
@@ -0,0 +1,39 @@
+import { Enum, Timestamp, UUID } from './utils'
+import { VEHICLE_STATE } from './vehicle/vehicle_states'
+import { VEHICLE_TYPE } from './vehicle/vehicle_types'
+
+export const PROPULSION_TYPES = Enum('human', 'electric', 'electric_assist', 'hybrid', 'combustion')
+export type PROPULSION_TYPE = keyof typeof PROPULSION_TYPES
+export const RIGHT_OF_WAY_STATES = ['available', 'reserved', 'non_operational', 'trip'] as const
+
+export const ACCESSIBILITY_OPTIONS = ['wheelchair_accessible'] as const
+export type ACCESSIBILITY_OPTION = typeof ACCESSIBILITY_OPTIONS[number]
```

Comment:
Just FYI, but you _can_ use the same name for both the `const` and `type` and TS will figure out which to use based on context. 
```ts
export const ACCESSIBILITY_OPTION = ['wheelchair_accessible'] as const
export type ACCESSIBILITY_OPTION = typeof ACCESSIBILITY_OPTION[number]
```